satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -730,3 +730,43 @@ read count (
 )
 
 create user_credit ()
+
+//--- metainfo buckets ---//
+
+model bucket (
+	key    id
+	unique name project_id
+
+	field id             blob
+	field project_id     project.id restrict
+	field name           blob
+
+	field path_cipher    int
+
+	field created_at     utimestamp ( autoinsert )
```

Comment:
Let's use `timestamp`, because apparently this avoids some weirdness of how databases handle timezones.